protoc_plugin/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 21.1.0-wip
+
+* Experimental support for generating comments. Use the `generate_comments` option. ([#161])
```

Comment:
Why do you say it's experimental? If there isn't anything unclear (maybe regarding how the `SourceCodeInfo` etc. are used) or unsupported I think we should just advertise this as "support".

Reply:
I'm just not very committal. We can call it "ready"

In fact, happy to discuss not putting it behind a flag.

Reply:
I checked a few of the protoc compilers used internally. Java and Go generate comments. C++ doesn't. AFAIK this library mostly copies Java (the lite version) so I say we generate comments always and remove the flag.